### DISCUSSION TITLE
[Feature Request] Simple StandardNotes to FileSend?

### DISCUSSION THREAD

**Discussion by redbernarr** [standardnotes/forum#2623] [Suggestions]
**Have you read the [Longevity Statement](https://standardnotes.org/longevity)?**

Yes! And I totally get it. It's why I pay! I understand that this may be a) not possible, b) not practical, or c) not worthwhile. It's just something that I thought *might* be significantly simpler than many extensions that exist. But I might just be flat wrong about that.

**Is your feature request related to a problem? Please describe.**
Trying to send text to others from Standard Notes while keeping the text encrypted both in transit and at rest is difficult. At the moment, my process is to copy the text, paste it into a .txt on my desktop, then upload the .txt to filesend, and send the link. Then I have to go back and delete the file from the desktop, and empty the trash. I work with a group that were using this process, but they've stopped because they have found it too difficult. 

**Describe the solution you'd like**
I would love a single button in StandardNotes that provides a FileSend link for a .txt of the note's contents (or even just opens the FileSend website, creates and uploads the necessary .txt and then deletes it from the hard drive, i don't know, y'all are much wiser than I am). ... I guess this is just as much a request about FileSend as it is about StandardNotes?

**Describe alternatives you've considered**
Well, I've laid out my normal process, and we've certainly considered using other tools, but the stability and security of standard notes is just unparalleled. We've considered trying to use filevault, but it turns out to be more clunky. Unfortunately, because the process I've outlined is so clunky, people have just given up on having things encrypted at rest and are using FileSend for various documents they just store unencrypted on their hard drives...which is fine, but it would be phenomenal to keep everything inside StandardNotes.

**Comment by atmoio** [comment]
Interesting idea! I agree that would be really handy. I'd have to think more about how we could accomplish it, but as a first thought, building a bridge between the two would seem to require sending unencrypted data to a server. Which is not ok. Currently with FileSend the server never sees unencrypted data. So we can't go the API route.

But nonetheless, definitely something to think about!

**Comment by cyb3rz3us** [comment]
Thought --- could public\private key encryption be used here?  So the cleartext is never sent...just the encrypted note is sent...

**Comment by atmoio** [comment]
Could be! We'd have to really get into it to determine the best option.
